netket/graph/abstract_graph.py

```diff
@@ -0,0 +1,47 @@
+import abc
+import numpy as _np
+
+
+class AbstractGraph(abc.ABC):
+    """Abstract class for NetKet graph objects"""
+
+    @property
+    @abc.abstractmethod
+    def is_connected(self):
+        r"""bool: True if the graph is connected"""
+        return NotImplementedError
```

Comment:
I think these should be raise NotImplementedError, also in the other files you have written

Reply:
> Thanks! we are getting very close! There is a few microscopic changes + please also try to install an automatic black linter for the code otherwise the tests won't pass!
> 
> ```
> 2.86s$ python -m black --check ../netket/
> would reformat /home/travis/build/netket/netket/netket/graph/hypergrid.py
> would reformat /home/travis/build/netket/netket/netket/graph/graph.py
> Oh no! 💥 💔 💥
> 2 files would be reformatted, 75 files would be left unchanged.
> ```

Oh!, I forgot about it completely, sorry. I have installed black already!